client/my-sites/plan-features/style.scss

```diff
@@ -95,7 +96,7 @@ $plan-features-sidebar-width: 272px;
 	border-left: solid 1px lighten( $gray, 27% );
 	background-color: lighten( $gray, 35% );
 	position: relative;
-	width: 300px;
+	width: 25%;
```

Comment:
Yeh, it can't be a fixed width. The other option is to leave out the width entirely. Let me test that.